src/test/java/com/datalogics/pdf/samples/extraction/TextExtractTest.java

```diff
@@ -87,7 +87,12 @@ public void testDocumentWithNoText() throws Exception {
         assertEquals(Level.INFO, logRecord.getLevel());
 
         // Verify that the output file was not created
-        final Path path = Paths.get(outputUrl.getPath());
+        /*
+         * appropriateOsPath: If the beginning of the string is a slash, then a character, then a colon and another
+         * slash, replace it with the character, the colon, and the slash (leaving the leading slash off).
+         */
+        final String appropriateOsPath = outputUrl.getPath().replaceFirst("^/(.:/)", "$1");
```

Comment:
mmmmm...this might fix it, but I'm kind of wary about fixes that try to target a platform. I'm thinking of doing my work in a directory called ``/n:myworkspace/...`` :smile: 

I think we discovered along the way that ``outputUrl.getPath()`` wasn't a good idea, but that ``outputUrl.toURI().getPath()`` got better results. We fixed other samples to use this pattern. Could you give that a try and see if it fixes a problem.